docs/user/openstack/README.md

```diff
@@ -501,6 +501,14 @@ spec:
           availabilityZone: <optional_openstack_availability_zone>
 ```
 
+[provider-spec-definition]: https://github.com/openshift/cluster-api-provider-openstack/blob/155384b859c5b2fb5b7f11c9111d3f8e4f3066bd/pkg/apis/openstackproviderconfig/v1alpha1/types.go#L31
+
+#### Defining a New MachineSet That Uses Multiple Networks
+
+To attach multiple networks to a machineSet, the `primarySubnet` value in the `providerSpec` must be set to the [machines subnet](./customization.md#cluster-scoped-properties). If this value is not set correctly, the nodes will be unable to join the cluster.
+
+Nodes' Kubernetes endpoints are published on the primary subnet. After you set the subnet, add all of the networks that you want to attach to your machines to the `Networks` list in `providerSpec`. You must also add the network that the primary subnet is part of to this list.
```

Comment:
Is ```Nodes'``` added on purpose?

Reply:
"The Kubernetes endpoints of nodes are published..." ?

Reply:
Yeah, The node's kubernetes endpoints are being published on the primary subnet